recirq/quantum_chess/circuit_transformer.py

```diff
@@ -133,10 +144,11 @@ def map_helper(self,
 
         Args:
           cur_node: node to examine.
-          mapping: current mapping of named qubits to `GridQubits`
-          available_qubits: current set of unassigned qubits
+          mapping: current mapping of named qubits to `GridQubits`,
+          available_qubits: current set of unassigned qubits,
           graph: adjacency graph of connections between qubits,
-            representing by a dictionary from qubit to adjacent qubits
+            representing by a dictionary from qubit to adjacent qubits,
+          nodes_trying: saves the current nodes under trying in this stack.
```

Comment:
This is awkward wording:  "saves the current nodes under trying in this stack"

Reply:
Now it's been re-phrased.

Reply:
Thanks that's re-phrased.